quax/_core.py

```diff
@@ -14,6 +14,7 @@
 import jax.tree_util as jtu
 import plum
 from jax.custom_derivatives import SymbolicZero as SZ
+from jax.extend.core import jaxpr_as_fun
```

Comment:
Ultranit: I tend to avoid `from foo import bar` imports most of the time. I'm actually a little surprised I already have this for the Symbolic Zero, normally I'd only do this for type hints like in jaxtyping!

Reply:
Done.